packages/react-vapor/src/components/multilineBox/hoc/MultilineBoxWithRemoveButton.tsx

```diff
@@ -73,8 +76,8 @@ export const multilineBoxWithRemoveButton = (
                     style={{
                         visibility: !data.isLast ? 'visible' : 'hidden',
                     }}
-                    onClick={() => !data.isLast && this.props.removeBox(data.id)}
-                    enabled={!data.isLast}
+                    onClick={() => !data.isLast && !this.props.disabled && this.props.removeBox(data.id)}
+                    enabled={!(this.props.disabled || data.isLast)}

```

Comment:
I'm a little confused about `!(this.props.disabled || data.isLast)` :joy: Could we extract `this.props.disabled && data.isLast` into a variable then we can use it in both line 65 and line 80? 
Or simply remove `={!(this.props.disabled || data.isLast)}`, I think `enabled` could be always `true` here, otherwise it returns already `null` at the line 66.